helm/generic-node/templates/generic-node-service.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ $.Values.name }}
+spec:
+  type: LoadBalancer
+  # We want all pods to have their addresses published for the sake of the
+  # other testnet pods even before they're ready, since they
+  # have to be able to talk to each other in order to become ready.
+  publishNotReadyAddresses: true
+  selector:
+    app: {{ $.Values.name }}
+  ports:
+  - name: tcp-p2p
+    port: {{ .Values.ports.p2p }}
+    targetPort: external-port
+    nodePort: 30005
```

Comment:
This should not be hardcoded, i feel like there will be port conflicts, but can it just be `{{ .Values.ports.p2p }}` ?
maybe {{ .Values.ports.p2p }} + 1000